### ISSUE TITLE
setup.py uses six, which hasnt been installed yet

### ISSUE THREAD

**Issue by jayvdb** [ar4s/flake8_tuple#3]
setup.py adds `six` as a dependency, but then uses six.  It isnt installed yet, and in a clean virtualenv it causes a failure like:

```
11:08:32 Collecting flake8-tuple>=0.2.4
11:08:32   Downloading flake8_tuple-0.2.5.tar.gz
11:08:32     Complete output from command python setup.py egg_info:
11:08:32     Traceback (most recent call last):
11:08:32       File "<string>", line 1, in <module>
11:08:32       File "/tmp/pip-build-KiKdik/flake8-tuple/setup.py", line 3, in <module>
11:08:32         import flake8_tuple
11:08:32       File "/tmp/pip-build-KiKdik/flake8-tuple/flake8_tuple.py", line 6, in <module>
11:08:32         import six
11:08:32     ImportError: No module named six
```

**Comment by ar4s** [comment]
fixed in 0.2.6

**Comment by jayvdb** [comment]
Installation now works
